zio-cli/shared/src/main/scala/zio/cli/Options.scala

```diff
@@ -378,6 +396,11 @@ object Options extends OptionsPlatformSpecific {
     }
 
     override val alternatives = List(left, right)
+
+    override val nameAndAliases: Set[String] = self.left.nameAndAliases ++ self.right.nameAndAliases
+
+    // TODO: fix this
+    override val valueCount: Int = 0
```

Comment:
Not sure what `valueCount` should be for `Options.OrElse`.
@jdegoes @adamgfraser @senia-psm

Reply:
`Options.Single` has a similar value called `names`. Maybe it is better to merge it with `nameAndAliases`.

Reply:
`nameAndAliases` now calls `names` for `Options.Single`. I'm keeping `nameAndAliases` as this is a new property in the `Opitons` trait. If you find the name too verbose, I can also make `names` a property in the trait instead. Thanks for the suggestion.

Reply:
Depends on what `valueCount` is used for, but changing it from an `Int` to a integral range would make it more compatible with `orElse`. Because alternations can produce `n` items where `n` is between `[min, max]`. Or you could as also, instead of ranges, be more precise with `NonEmptySet[Int]`, which would describe all possibilities in every sum.

Reply:
Fixed. I changed the implementation a bit, for the better, I think. Please review. Thanks.